packages/panna-sdk/src/core/utils/activity.ts

```diff
@@ -224,12 +227,11 @@ export const getActivitiesByAddress = async function (
   }
 
   const activities: Activity[] = userTransactions
-    .slice(offset, offset + limit)
     .map((tx) => {
       const transactionID = tx.hash;
       const status = tx.result;
 
-      let activityType: ActivityType = TransactionActivity.SENT;
+      let activityType: ActivityType = TransactionActivity.UNKNOWN;
```

Comment:
The downstream logic may not handle this case properly, potentially causing runtime errors.
Perhaps we add explicit handling for unknown types in the switch statement or return early?
```typescript
if (activityType === TransactionActivity.UNKNOWN) {
  console.warn(`Unknown transaction type for tx ${transactionID}, skipping`);
  return null; // Skip unknown transactions gracefully
}
```